api/v1alpha1/hostedcluster_types.go

```diff
@@ -538,6 +547,111 @@ type IBMCloudPlatformSpec struct {
 	ProviderType configv1.IBMCloudProviderType `json:"providerType,omitempty"`
 }
 
+// PowerVSPlatformSpec defines IBMCloud PowerVS specific settings for components
+type PowerVSPlatformSpec struct {
+	// ResourceGroup is the IBMCloud Resource Group in which the cluster resides. If not
+	// specified then default resource group of the account will be used.
+	//
+	// +immutable
+	ResourceGroup string `json:"resourceGroup,omitempty"`
+
+	// Region is the IBMCloud region in which the cluster resides. This configures the
+	// OCP control plane cloud integrations, and is used by NodePool to resolve
+	// the correct boot image for a given release.
+	//
+	// +immutable
+	Region string `json:"region"`
+
+	// Zone is the availability zone where control plane cloud resources are
+	// created.
+	//
+	// +immutable
```

Comment:
there's omitempty here, can we add +optional tag too consistently everywhere is optional?